scripts/configdb_readxtc

```diff
@@ -1,46 +1,46 @@
 #! /bin/bash
 usage()
 {
-cat << EOF
+    cat << EOF
 usage: $0 options
 
 We will run configdb_readxtc
 
 OPTIONS:
 -u user (needs to be able to log into the psananeh/feh)
--e expnumber 
+-e expnumber
 EOF
 }
 
 if [[ ($1 == "--help") || ($1 == "-h") ]]; then
-	usage
-	exit 0
+    usage
+    exit 0
 fi
 
-USER=`whoami`
-ARGSTR=''
+USER=$(whoami)
+export ARGSTR=''
```

Comment:
why do we export this variable and not the other?  This also assumes we don't source this, otherwise we get `ARGSTR` in our environment variables right?

Reply:
I was too lazy to check if the child process this spawned needed the variables to be set or the empty string specifically so left it in case it was used, and exported it to get rid of the shellcheck warning. Searching through https://github.com/lcls-daq/pdsapp, I don't see ARGSTR used anywhere so it probably should be removed.

Reply:
I went through the exports I added and removed most of them. In the cases where I think the variables are completely unused, I deleted them. In others I excused the warning. Specifically,

- Removing NOTRUNNING from wherepsana
- removing STOP_AMI from startami
- Removing test and and if block in restartdaq that does nothing
- Removing the export and disabling warnings on STARTTIME and ENDTIME in motorinfo since a comment suggests they may be used in the future
- Removing ARGSTR from configdb_readxtc
- Removing the export and disabling the warning on S3DF=1 in makepeds. I can't tell what BINPATH resolves to from reading the script so I didn't remove the variable entirely